openadapt/app/dashboard/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Can you please clarify where this license is from?

Reply:
Removed this license, this came in with the starter template